spec/services/idv/duplicate_ssn_finder_spec.rb

```diff
@@ -24,6 +24,26 @@
 
         expect(subject.ssn_is_unique?).to eq false
       end
+
+      it 'recognizes fingerprint without dashes' do
+        create(:profile, pii: { ssn: ssn.delete('-') })
```

Comment:
noob question, does `create(:profile)` generate a test SSN? I notice that we don't define the test `123456789` as in the other cases below

Reply:
It's due to the `let` block at the top which defines `ssn`, which the `subject` uses to check for unique PII. For consistency, I've switched this one to be in the same format as the cases below.